cmd/signozschemamigrator/schema_migrator/traces_migrations.go

```diff
@@ -1,3 +1,417 @@
+					{Name: "httpHost", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "http_host"},
+					{Name: "dbName", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "db_name"},
+					{Name: "dbOperation", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "db_operation"},
+					{Name: "hasError", Type: ColumnTypeBool, Alias: "has_error"},
+					{Name: "isRemote", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "is_remote"},
+
+					{Name: "serviceName", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "resource_string_service$$name"},
+
+					// ALIAS for simple attrs
+					{Name: "httpRoute", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_http$$route"},
+					{Name: "msgSystem", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_messaging$$system"},
+					{Name: "msgOperation", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_messaging$$operation"},
+					{Name: "dbSystem", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_db$$system"},
+					{Name: "rpcSystem", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_rpc$$system"},
+					{Name: "rpcService", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_rpc$$service"},
+					{Name: "rpcMethod", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_rpc$$method"},
+					{Name: "peerService", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "attribute_string_peer$$service"},
+				},
+				Indexes: []Index{
+					{Name: "idx_trace_id", Expression: "trace_id", Type: "tokenbf_v1(10000, 5,0)", Granularity: 1},
+					{Name: "idx_spanID", Expression: "span_id", Type: "tokenbf_v1(5000, 5,0)", Granularity: 1},
+					{Name: "idx_duration", Expression: "duration_nano", Type: "minmax", Granularity: 1},
+					{Name: "idx_name", Expression: "name", Type: "ngrambf_v1(4, 5000, 2, 0)", Granularity: 1},
+					{Name: "idx_kind", Expression: "kind", Type: "minmax", Granularity: 4},
+					{Name: "idx_httpRoute", Expression: "attribute_string_http$$route", Type: "bloom_filter", Granularity: 4},
+					{Name: "idx_httpUrl", Expression: "http_url", Type: "bloom_filter", Granularity: 4},
+					{Name: "idx_httpHost", Expression: "http_host", Type: "bloom_filter", Granularity: 4},
+					{Name: "idx_httpMethod", Expression: "http_method", Type: "bloom_filter", Granularity: 4},
+					{Name: "idx_timestamp", Expression: "timestamp", Type: "minmax", Granularity: 1},
+					{Name: "idx_rpcMethod", Expression: "attribute_string_rpc$$method", Type: "bloom_filter", Granularity: 4},
+					{Name: "idx_responseStatusCode", Expression: "response_status_code", Type: "set(0)", Granularity: 1},
+					{Name: "idx_statusCodeString", Expression: "status_code_string", Type: "set(3)", Granularity: 4},
+					{Name: "idx_spanKind", Expression: "span_kind", Type: "set(5)", Granularity: 4},
```

Comment:
nit: Please update the expression names also.

Reply:
Done